cmd/codegen.go

```diff
@@ -0,0 +1,62 @@
+package cmd
+
+import (
+	"os"
+	"os/exec"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+const DirPerms = 0700
+
+func InstallCodegen(dir string) error {
+	prompt := &survey.Confirm{
+		Message: "Do you want to install the TypeScript SDK and code generator?",
+		Default: true,
+	}
+	var install bool
+	if err := survey.AskOne(prompt, &install); err != nil {
+		return err
+	}
+
+	if !install {
+		return nil
+	}
+
+	npm, err := exec.LookPath("npm")
+	if err != nil {
+		return err
+	}
+
+	if err = execNpm(npm, "install", "@xata.io/client"); err != nil {
+		return err
+	}
+
+	if err = execNpm(npm, "install", "@xata.io/codegen", "-D"); err != nil {
+		return err
+	}
+
+	settings, err := ReadSettings(dir)
+	if err != nil {
+		return err
+	}
+
+	settings.Hooks["build"] = "./node_modules/.bin/xata-codegen generate -o client.ts"
```

Comment:
```suggestion
	settings.Hooks["build"] = "./node_modules/.bin/xata-codegen -o client.ts"
```

We'll change this today probably https://github.com/xataio/client-ts/pull/111